src/daml/Daml/Trading/Account/Account.daml

```diff
@@ -33,14 +38,16 @@ template HoldingFactoryReference
 
 template Request 
   with 
-    custodian: Party 
+    custodian: Party
     owner: Party 
     accountType: AccountType 
     username: Text 
     realName: Text 
     passportNumber: Text 
     nationalId: Text 
     address: Text 
+    rewardPoint : Int
+    badges: [BadgeType]
```

Comment:
we don't need `rewardPoint` and `badges` in Request because for a new user `rewardPoint` will always be 0 and `badges` will always be empty